contracts/Enigma.sol

```diff
@@ -137,6 +140,24 @@ contract Enigma is EnigmaStorage, EnigmaEvents, Getters {
         _;
     }
 
+    /**
+    * Ensure that the contract's context is the updated contract
+    *
+    */
+    modifier isUpdatedEnigmaContract() {
+        require(address(this) == state.updatedEnigmaContractAddress, "Not updated Enigma contract");
+        _;
```

Comment:
I'm curious: What is this line for? `_;`

Reply:
modifier syntax is to use `_` as a placeholder as the final line of the modifier declaration, since the _remaining_ function for which the modifier applies to will be used there